lib/workload/components/api-gateway/index.ts

```diff
@@ -1,24 +1,26 @@
 import { Construct } from 'constructs';
 import { aws_ssm, Duration } from 'aws-cdk-lib';
 import { HttpJwtAuthorizer } from 'aws-cdk-lib/aws-apigatewayv2-authorizers';
-import { CorsHttpMethod, HttpApi } from 'aws-cdk-lib/aws-apigatewayv2';
+import { CorsHttpMethod, HttpApi, CfnStage } from 'aws-cdk-lib/aws-apigatewayv2';
 import { IStringParameter } from 'aws-cdk-lib/aws-ssm';
+import { LogGroup } from 'aws-cdk-lib/aws-logs';
+import { Role, ServicePrincipal } from 'aws-cdk-lib/aws-iam';
 
-export interface SRMApiGatewayConstructProps {
+export interface ApiGatewayConstructProps {
   region: string;
   cognitoUserPoolIdParameterName: string;
   cognitoPortalAppClientIdParameterName: string;
   cognitoStatusPageAppClientIdParameterName: string;
 }
 
-export class SRMApiGatewayConstruct extends Construct {
+export class ApiGatewayConstruct extends Construct {
   private readonly _httpApi: HttpApi;
 
-  constructor(scope: Construct, id: string, props: SRMApiGatewayConstructProps) {
+  constructor(scope: Construct, id: string, props: ApiGatewayConstructProps) {
     super(scope, id);
 
     this._httpApi = new HttpApi(this, 'HttpApi', {
-      apiName: 'OrcaBusAPI-SequenceRunManager',
+      apiName: 'OrcaBusAPI-' + id,
```

Comment:
Can we expose this `apiName` prop as `ApiGatewayConstructProps` and, allow set by the caller.

This CDK construct ID behaviour is context specific depends on its parent stack.

The prefix `OrcaBusAPI-` is fine, still. 

i.e.

```
apiName: 'OrcaBusAPI-' + props.apiName,
```

And expose `apiName` prop at the above interface `ApiGatewayConstructProps`.